config/a8c-for-agencies-development.json

```
@@ -65,7 +66,8 @@
 		"a8c-for-agencies-migrations": true,
 		"a8c-for-agencies-client": true,
 		"a8c-for-agencies-team": true,
-		"a8c-for-agencies-agency-tier": true
+		"a8c-for-agencies-agency-tier": true,
+		"a8c-for-agencies-plugins-management": true
```

Let's keep the section name same as this: https://github.com/Automattic/wp-calypso/blob/b6e8cbf948c62c012743a92158f73c61990aa3d8/client/sections.js#L806

```suggestion
		"a8c-for-agencies-plugins": true
```

Also, let's add an item here: https://github.com/Automattic/wp-calypso/blob/b6e8cbf948c62c012743a92158f73c61990aa3d8/config/_shared.json#L160-L161. This is to make sure this won't cause any issues on WPCOM as it has done in the past.

```a8c-for-agencies-plugins: false```